packages/lib/telemetry.ts

```diff
@@ -35,24 +36,28 @@ export function collectPageParameters(
 }
 
 const reportUsage: EventHandler = async (event, { fetch }) => {
-  const ets = telemetryEventTypes;
-  if ([ets.bookingConfirmed, ets.embedBookingConfirmed].includes(event.eventType)) {
-    const key = process.env.CALCOM_LICENSE_KEY;
-    const url = `${CONSOLE_URL}/api/deployments/usage?key=${key}&quantity=1`;
-    try {
-      return fetch(url, { method: "POST", mode: "cors" });
-    } catch (e) {
-      console.error(`Error reporting booking for key: '${key}'`, e);
+  const session = await getSession();
```

Comment:
Why use session instead of prisma here? AFAIK `getSession` makes a request as well so there's no fetching benefit here

Reply:
I tried, but this code seems to be executed client side. I could rely on tRPC perhaps, or we can just remove the license key from telemetry all-together to avoid relying on session.